netctl/commands.go

```diff
@@ -168,27 +174,37 @@ var Commands = []cli.Command{
 				Name:      "ls",
 				Aliases:   []string{"list"},
 				Usage:     "List rules for a given tenant/policy",
-				ArgsUsage: "[policy]",
-				Flags:     []cli.Flag{tenantFlag, allFlag, jsonFlag},
-				Action:    listRules,
+				ArgsUsage: "",
+				Flags: []cli.Flag{tenantFlag, allFlag, jsonFlag, quietFlag,
+					cli.StringFlag{
+						Name:  "policy, p",
+						Usage: "Policy - REQUIRED",
+					},
+				},
+				Action: listRules,
 			},
 			{
 				Name:      "rm",
 				Aliases:   []string{"delete"},
 				Usage:     "Delete an existing rule.",
-				ArgsUsage: "[policy] [rule id]",
-				Flags:     []cli.Flag{tenantFlag},
-				Action:    deleteRule,
+				ArgsUsage: "[rule id]",
```

Comment:
@vijayvikrant 
I can see that `netctl policy <policy-name> add-rule <rule-id> [rule-options]` might have been a better option. I was just basing it on `docker network connect` which takes two arguments in this form:
```
$ docker network connect --help

Usage:	docker network connect [OPTIONS] NETWORK CONTAINER

Connects a container to a network

  --help=false       Print usage
```
There are other example too like `docker run` takes an image-name and a command argument. We can follow those example i think.